src/explore-education-statistics-frontend/src/middleware/withMethods.ts

```diff
@@ -13,8 +13,8 @@ export type MethodsConfig = {
   [key in Method]?: NextApiHandler;
 };
 
-export default function withMethods(config: MethodsConfig): NextApiHandler {
```

Comment:
Not a big deal, but why have we changed this? This is the correct return type and dropping down to use inference isn't really preferable.

Reply:
Looks like the types have changed: 
```
Type '(req: NextApiRequest, res: NextApiResponse<any>) => Promise<void | NextApiResponse<any>>' is not assignable to type 'NextApiHandler'.
  Type 'Promise<void | NextApiResponse<any>>' is not assignable to type 'void | Promise<void>'.
    Type 'Promise<void | NextApiResponse<any>>' is not assignable to type 'Promise<void>'.
      Type 'void | NextApiResponse<any>' is not assignable to type 'void'.
        Type 'NextApiResponse<any>' is not assignable to type 'void'.ts(2322)
```